development/getting_code_on_the_robot.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
This page should definitely live under "Programming" rather than a new "Development" section